stackle_api/app/routes.js

```diff
@@ -12,198 +12,213 @@ var Reply = postModels.Reply;
-		res.header("Access-Control-Allow-Origin", "*");
-		res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
-		var orgname = req.params.orgname;
-		Stack.find({ name: orgname }, function (err, org) {
-			if (err) {
-				console.log('Error');
-			} else {
-				res.send(org);
-			}
-		})
-	})
-
-	//comment on a post
-	app.post('/api/comment/:postid', function (req, res) {
-		var postid = req.params.postid;
-		var query = { _id: postid };
-		var comment = new Comment(req.body);
-		Post.update(query, { comments: [] });
-	})
-
-	//get all stacks (orgs)
-	app.get('/api/orgs', function (req, res) {
-		Stack.find({}, function (err, stacks) {
-			if (err)
-				console.log("Errors retrieving stacks!");
-			else
-				res.send(stacks);
-		})
-	})
-
-	//create stack
-	app.post('/api/stack/create', function (req, res) {
-		var stack = new Stack(req.body);
-		stack.save(function (err, stack) {
-			if (err) {
-				console.log("Error saving the stack to database");
-				res.send("Error saving stack!");
-			} else if (stack) {
-				res.send("Sucessfully created the stack");
-			} else {
-				res.send("Null");
-			}
-		})
-	})
-
-	//delete stack
-	app.delete('api/delete/stack/:stackid', function (req, res) {
-		var stack_id = req.params.stackid;
-		Stack.remove({ _id: stack_id }, function (err, success) {
-			if (err) {
-				res.send("Couldn't delete Stack");
-			} else {
-				res.send("")
-			}
-		})
-	})
-
-	//user subscribing to an stack
-	app.post('/api/subscribe', function (req, res) {
-		var userid = req.body.uid;
-		var stackname = req.body.stack_name;
-		var query = { userId : userid };
-		User.findOneAndUpdate(query, {$push: {subscribed_stacks : stackname}}, function(err, noaffected){
-			if(err){
-				res.send("Error Updating");
-			}else{
-				res.send("Success!!");
-			}
-		});
-	})
-
-	//getting subscribed stacks for a user
-	app.get('/api/stack/subscribed/:userid', function(req ,res){
-		res.header("Access-Control-Allow-Origin", "*");
-		res.header('Access-Control-Allow-Methods: GET, POST, PATCH, PUT, DELETE, OPTIONS');
-		res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
-		User.findOne({userId : req.params.userid}, function(err, result){
-			if(err){
-				res.send(err);
-			}else if(result){
-				var sub_stack = result.subscribed_stacks;
-				res.send(sub_stack);
-			}else{
-				res.send("Can't get!");
-			}
-		})
-	})
-
-	//create user
-	app.post('/api/newuser', function (req, res) {
-		var user = new User(req.body);
-		user.save(function (err, user) {
-			if (err) {
-				console.log("Error saving the stack to database");
-				res.send("Error saving user!");
-			} else {
-				res.send("Sucessfully created the user");
-			}
-		})
-	})
-
-	app.get('/api/notifications', function (req, res) { });
-
-	app.get('/*', function (req, res) {
-		res.sendfile('./public/404.html');
-	});
+    //api
+    app.get('/api/login/', function (req, res) {
```

Comment:
Don't mind the Quotes .. though codacy points it out .. but anyway use Double Quotes to pass the codacy test

Reply:
I have created PR #89 to solve the codacy issues.